shipment.go

```diff
@@ -356,10 +375,16 @@ func (c *Client) LowestShipmentRate(shipment *Shipment) (out Rate, err error) {
 	return c.LowestShipmentRateWithCarrier(shipment, nil)
 }
 
+// LowestStatelessRate gets the lowest stateless rate from a list of stateless rates
+func (c *Client) LowestStatelessRate(rates []*StatelessRate) (out StatelessRate, err error) {
```

Comment:
This function needs to be in the `rate.go` file please since it has nothing to do with a shipment and everything to do with rates.